### DISCUSSION TITLE
Channel mixer - Converting 2 to 6 channels not implemented.

### DISCUSSION THREAD

**Discussion by Raul824** [audacious-media-player/audacious#1406] [General]
Hi All,

Am using audacious with output plugin SDL outpu as it is providing most multichannel audio support.
But when I am enabling channel mixer and setting it to 6 channel it says 
Conversion from 2 channel to 6 channel not implemented.
Could anyone please let me know where to implement this channel up-mixing.

Thanks.

**Comment by radioactiveman** [comment]
> Could anyone please let me know where to implement this channel up-mixing.

My suggestions would be to configure this globally in your audio framework like PulseAudio, PipeWire etc. (assuming you use Linux).
And not in applications like Audacious which make use of them.

Supported conversions by Audacious' channel mixer plugin can be seen here:
https://github.com/audacious-media-player/audacious-plugins/blob/beaf8fc27d0023136bb07d1b3c18278e460759c5/src/mixer/mixer.cc#L189-L205
